### PR TITLE
compensate scrollBarGap with body's own padding

### PR DESCRIPTION
**Pull request by AndreiEres**

### CODE REVIEW

**Comment by AndreiEres** [comment]
@willmcpo
Hi!
Shall we deal with this fix? :-)

File: src/bodyScrollLock.js [hunk 1]
```diff
@@ -77,8 +77,10 @@ const setOverflowHidden = (options?: BodyScrollOptions) => {
       const scrollBarGap = window.innerWidth - document.documentElement.clientWidth;
 
       if (reserveScrollBarGap && scrollBarGap > 0) {
+        const computedBodyPaddingRight = getComputedStyle(document.body).getPropertyValue('padding-right');
```
Comment (jossmac) [inline src/bodyScrollLock.js]: The call to `getPropertyValue` will return a string, so you'll get something like `"0px15px"`
```suggestion
        const computedBodyPaddingRight = parseInt(getComputedStyle(document.body).getPropertyValue('padding-right'), 10);
```

**Comment by AndreiEres** [comment]
@jossmac, thanks

**Comment by AndreiEres** [comment]
@willmcpo 
Could you please have a look?

**Comment by diachedelic** [comment]
Thanks @AndreiEres , I'm sorry that took so long. It will be in the next version @willmcpo publishes.

**Pull request merged**